Source/EasyNetQ/RequestConfiguration.cs

```diff
@@ -32,6 +32,13 @@ public interface IRequestConfiguration
     /// <returns>Returns a reference to itself</returns>
     IRequestConfiguration WithQueueName(string queueName);
 
+    /// <summary>
+    /// Sets the queue type. Valid types are "classic" and "quorum". Works with RabbitMQ version 3.8+.
+    /// </summary>
+    /// <param name="queueType">Desired queue type.</param>
+    /// <returns>Returns a reference to itself</returns>
+    IRequestConfiguration WithQueueType(string queueType);
```

Comment:
```suggestion
    IRequestConfiguration WithQueueType(string queueType = QueueType.Classic);
```

Reply:
just to align with `IReceiveConfiguration WithQueueType(string queueType = QueueType.Classic);`

Reply:
Fixed.